### DISCUSSION TITLE
Can I disable the aliases "~~", "@", "@@" e.t.c.?

### DISCUSSION THREAD

**Discussion by szulcus** [nuxt/nuxt#16475] [Questions] [answered]
The editor sometimes plays tricks on me and imports files with the "~~" prefix which contaminates my code.

Based on the [documentation](https://v3.nuxtjs.org/api/configuration/nuxt-config/), I tried to configure nuxt like this, but it didn't do anything:
```ts
alias: {
	'~': path.resolve(__dirname),
},
```
Is it possible to disable these aliases?

**Comment by danielroe** [comment, marked as answer]
https://github.com/nuxt/framework/issues/7277#issuecomment-1252390503

**Reply by szulcus** [reply]
Amazing hook, thanks!
